src/coreclr/jit/lower.cpp

```diff
@@ -5447,9 +5453,13 @@ void Lowering::LowerAsyncContinuation(GenTree* asyncCont)
                 node->AsCall()->gtIsAsyncCall = true;
             }
 
+            BlockRange().Remove(asyncCont);
+            BlockRange().InsertAfter(node, asyncCont);
```

Comment:
If we go backwards from continuation to find the call, why we need to force the order? Would it be "after" anyways?"

Reply:
I see - to make sure there is nothing in between.